README.md

```diff
@@ -24,7 +24,8 @@ Add `storage` block to file `config.js` in each environment as below:
 			active: 'ghost-azurestorage',
 			'ghost-azurestorage': {
 				connectionString: 'YourConnectionStringHere',
-        container: 'YourOptionalContainerName'
+				container: 'YourOptionalContainerName',
+				cdnUrl: "YourAzureCDNEndpointDomain"
```

Comment:
Thanks for including doc updates to this!

Perhaps you should just put YourCDNEndpointDomain... adding the word azure would imply that it couldn't work with any other CDN's which is obv. not true